lib/src/widgets/typing/ds_dot_animation.widget.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter/material.dart';
+
+import 'ds_dot.widget.dart';
+
+/// To define the stitch appearance use [size] and [color]
+/// Use [numberOfDots] to set the amount of dots and use [innerPadding] to set the spacing between them
+/// To change the animation speed use [animationTime]
+class DsDotAnimationWidget extends StatefulWidget {
```

Comment:
Use the prefix 'DS' in uppercase, without widget in the class name..

Could move this widget into widgets/chat folder

Reply:
Done